daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -127,3 +127,19 @@ message NodeRollback { // *since version 14*
 }
 
 // architecture-handbook-entry-end: Nodes
+
+message Versioned {
```

Comment:
I don't know exactly where we document the LF proto messages, but it looks like this PR is missing the docs for how these messages and fields are intended to be used. Let's add that :)

Reply:
I added a section in daml-lf/spec/transaction.rst which document the `transation.proto` format.

Reply:
added a section in the transaction speicification